packages/monosize/src/commands/measure.mts

```diff
@@ -39,11 +40,17 @@ async function measure(options: MeasureOptions) {
     console.log(`${pc.blue('[i]')} artifacts dir is cleared`);
   }
 
-  const fixtures = glob.sync('bundle-size/*.fixture.js', {
+  const fixtureFilesGlob = fixturesGlob ? fixturesGlob : '*.fixture.js'; 
```

Comment:
with CLI command properties defined as requested in other comment

this can be simplified to following:

```ts
const fixtures = glob.sync(`bundle-size/${fixturesGlob}`, {
    absolute: true,
    cwd: process.cwd(),
  });

  if (!fixtures.length) {
```